staging/src/k8s.io/cli-runtime/pkg/printers/managedfields.go

```diff
@@ -44,8 +45,9 @@ func omitManagedFields(o runtime.Object) runtime.Object {
 // PrintObj copies the object and omits the managed fields from the copied object before printing it.
 func (p *OmitManagedFieldsPrinter) PrintObj(obj runtime.Object, w io.Writer) error {
 	if obj == nil {
-		return p.Delegate.PrintObj(obj, w)
+		return fmt.Errorf("error: unable to find the specified revision")
```

Comment:
I'm not sure about this change for a couple of reasons:
1. This printer is used by many other resource types, so we can't return an error about the revision not being found... it wouldn't make sense if it was a different resource type.
2. If you look at panic's call stack, you can see that the panic occurs in the delegate printer, so for example the YAMLPrinter or JSONPrinter.  Should those printers be the ones to actually check if the object is nil and return an error?  It might be a good change, but I don't know if it needs to be part of this PR though.